test/benchmark_extended_jump_array.jl

```diff
@@ -0,0 +1,27 @@
+# This file is not directly included in a test case, but is used to
+# benchmark and compare changes to the broadcsting
+using JumpProcesses, StableRNGs, FastBroadcast, BenchmarkTools
```

Comment:
Move it to a benchmarks folder at the top level like in https://github.com/SciML/LinearSolve.jl